common/updates-model.cpp

```diff
@@ -84,20 +88,29 @@ void updates_model::draw(std::shared_ptr<notifications_model> not_model, ux_wind
             // Draw Firmware update Pane
             // ===========================================================================
             fw_update_needed = draw_firmware_section(not_model, window_name, update, positions, window, error_message);
-
         }
         else 
-        {
-            ImGui::PushFont(window.get_large_font());
+        { // Indicate device disconnected to the user
             ImGui::PushStyleColor(ImGuiCol_Text, white);
-            ImGui::SetCursorPos({ positions.orig_pos.x, positions.y0 - 100 });
             ImGui::SetWindowFontScale(1.5);
-            ImGui::Text("%s","THE DEVICE HAS BEEN DISCONNECTED,");
-            ImGui::SetCursorPos({ positions.orig_pos.x - 100, positions.y0 - 70 });
-            ImGui::Text("%s", "PLEASE RECONNECT IT OR CLOSE THE UPDATES WINDOW.");
-            ImGui::PopFont();
-            ImGui::SetCursorPos({ positions.orig_pos.x + 230, positions.y0 });
+            std::string disconnected_text = "THE DEVICE HAS BEEN DISCONNECTED,";
```

Comment:
I don't think the punctuation is needed -- in the next line, too
You have a picture or sample to show me?